<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="DrawLabel" name="labelBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1028</width>
      <height>597</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Background.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_ALibrary1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Library.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Gym18">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Gym.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Dorm313">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>315</y>
      <width>18</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Dormitory3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Dorm414">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Dormitory4.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Dorm616">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>17</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Dormitory6.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Bathroom2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>14</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Bathroom.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Dorm515">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>167</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Dormitory5.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Dorm212">
    <property name="geometry">
     <rect>
      <x>442</x>
      <y>225</y>
      <width>15</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Dormitory2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Dorm111">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>17</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Dormitory1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_DiningHall8">
    <property name="geometry">
     <rect>
      <x>407</x>
      <y>397</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/DiningHall.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Door19">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>312</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Door.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Office19">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Office.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Block57">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Block5.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Block46">
    <property name="geometry">
     <rect>
      <x>373</x>
      <y>220</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Block4.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Block35">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>188</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Block3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_TJiaoxue22">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>215</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/School.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_SJiaoyan21">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Science.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Experiment17">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>14</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/ExperimentBuilding.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Door210">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>250</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Door.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_RKeyan20">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>16</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Technology.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Block13">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>165</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Block1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyButton" name="pushButton_Block24">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>168</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Block2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frameSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="textEditSearch">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>5</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kaiti SC</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/TitleImage/Background2.jpg);</string>
     </property>
     <property name="tabStopWidth">
      <number>84</number>
     </property>
     <property name="placeholderText">
      <string>请输入搜索位置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_22">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>39</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/TitleImage/CampusTourGuideLogo.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_fang_da">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>5</y>
       <width>28</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/TitleImage/PButtonSearch.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Clear">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/SearchImage/SearchCancel.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonPoints">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/TitleImage/Background2.jpg);
font: 75 13pt &quot;Kaiti SC&quot;;</string>
       </property>
       <property name="text">
        <string>多个点生成树</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/TitleImage/PButtonMap.png</normaloff>:/image/TitleImage/PButtonMap.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonShortest">
       <property name="styleSheet">
        <string notr="true">font: 13pt &quot;Kaiti SC&quot;;
border-image: url(:/image/TitleImage/Background2.jpg);</string>
       </property>
       <property name="text">
        <string>最短路径</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/TitleImage/PButtonShortest.png</normaloff>:/image/TitleImage/PButtonShortest.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAll">
       <property name="styleSheet">
        <string notr="true">font: 13pt &quot;Kaiti SC&quot;;
border-image: url(:/image/TitleImage/Background2.jpg);</string>
       </property>
       <property name="text">
        <string>全部路径</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/TitleImage/PButtonAll.png</normaloff>:/image/TitleImage/PButtonAll.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAir">
       <property name="styleSheet">
        <string notr="true">font: 13pt &quot;Kaiti SC&quot;;
border-image: url(:/image/TitleImage/Background2.jpg);</string>
       </property>
       <property name="text">
        <string>空气质量</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/TitleImage/PButtonAir.png</normaloff>:/image/TitleImage/PButtonAir.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="loginStart">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/User.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frameLogin">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>70</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/FrameImage/FrameLogin.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="loginCloseButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>14</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/TitleImage/PButtonFLoginCancel.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>13</y>
       <width>61</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Kaiti SC</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/FrameImage/LoginNext.png);
font: 13pt &quot;Kaiti SC&quot;;</string>
     </property>
     <property name="text">
      <string>登陆</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_26">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>68</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_27">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>103</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="DescLabel" name="infoPanel" native="true">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>390</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 231, 231);</string>
    </property>
    <widget class="QTextEdit" name="buttonName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>42</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;NULL&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>编号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="buttonID">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>0</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>21</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>名称：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>73</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>简介：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="buttonInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
    </widget>
    <widget class="QWidget" name="adminPanel" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>-1</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QPushButton" name="saveChange">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteNode">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frameSearch_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/SearchImage/backgroundSearch.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="button_go">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/SearchImage/ButtonByfoot.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>8</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/SearchImage/Byfoot.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>60</y>
       <width>165</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/SearchImage/TextEditAdd.png);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>102</y>
       <width>165</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/SearchImage/TextEditAdd.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search2_closeButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/SearchImage/SearchNextCancel.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="multiPointConfirm">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>47</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header>mybutton.h</header>
  </customwidget>
  <customwidget>
   <class>DrawLabel</class>
   <extends>QLabel</extends>
   <header>DrawLabel.h</header>
  </customwidget>
  <customwidget>
   <class>DescLabel</class>
   <extends>QWidget</extends>
   <header>desclabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
